services/github/github-package-json.service.js

```diff
@@ -23,8 +23,7 @@ class GithubPackageJsonVersion extends ConditionalGithubAuthService {
   static get route() {
     return {
       base: 'github/package-json/v',
-      format: '([^/]+)/([^/]+)/?([^/]+)?',
-      capture: ['user', 'repo', 'branch'],
+      pattern: ':user/:repo/:branch?',
```

Comment:
Ditto on example error here

Reply:
Did you mean to add something like **(branch)** to the title? I've just gone ahead and done that.

Reply:
Sorry I should've been more clear. I meant that with this badge is showing an `invalid response data` error with the example too (also occurs in prod so not related to these changes)